docs/docs-beta/docs/dagster-plus/deployment/management/managing-compute-logs-and-error-messages.md

```diff
@@ -3,4 +3,106 @@ title: Managing compute logs and error messages
 unlisted: true
 ---
 
-{/* TODO move from https://docs.dagster.io/dagster-plus/managing-deployments/controlling-logs */}
\ No newline at end of file
+import ThemedImage from '@theme/ThemedImage';
+
+:::note
+This guide is applicable to Dagster+.
+:::
+
+In this guide, we'll cover how to adjust where Dagster+ compute logs are stored and manage masking of error messages in the Dagster+ UI.
+
+{/* By default, Dagster+ ingests [structured event logs and compute logs](/concepts/logging#log-types) from runs and surfaces error messages from [code locations](/guides/deploy/code-locations/) in the UI. */}
+By default, Dagster+ ingests [structured event logs and compute logs](/todo) from runs and surfaces error messages from [code locations](/todo) in the UI.
+
+Depending on your organization's needs, you may want to retain these logs in your own infrastructure or mask error message contents.
+
+---
+
+## Modifying compute log storage
+
+Dagster's compute logs are handled by the configured [`ComputeLogManager`](/api/python-api/internals#compute-log-manager). By default, Dagster+ utilizes the `CloudComputeLogManager` which stores logs in a Dagster+-managed Amazon S3 bucket, but you can customize this behavior to store logs in a destination of your choice.
+
+### Writing to your own S3 bucket
+
+If using the Kubernetes agent, you can instead forward logs to your own S3 bucket by using the [`S3ComputeLogManager`](/api/python-api/libraries/dagster-aws#dagster_aws.s3.S3ComputeLogManager).
+
+{/* You can configure the `S3ComputeLogManager` in your [`dagster.yaml` file](/dagster-plus/deployment/agents/customizing-configuration): */}
+You can configure the `S3ComputeLogManager` in your [`dagster.yaml` file](/todo):
+
+```yaml
```

Comment:
Do yaml snippets need to be included in `/examples/docs_beta_snippets`?

Reply:
I think it's fine to keep them embedded for now.

Eventually it would be a good practice to go through all code snippets and determine which would benefit from moving to `docs_beta_snippets` -- the primary benefit is testing / linting.